web/src/settings_emoji.ts

```diff
@@ -63,11 +65,12 @@ export function update_custom_emoji_ui() {
     populate_emoji();
 }
 
-export function reset() {
+export function reset(): void {
     meta.loaded = false;
 }
 
-function sort_author_full_name(a, b) {
+function sort_author_full_name(a: ServerEmoji, b: ServerEmoji): number {
+    assert(a.author && b.author, "Both authors should be present.");
```

Comment:
Replied at https://chat.zulip.org/#narrow/stream/6-frontend/topic/null.20author.20id.20for.20custom.20emojis.3F. Because `null` is possible, using `assert` would crash the webapp. Let's do some additional checks here.